### ISSUE TITLE
Emagged borgs are easier then ever to discover

### ISSUE THREAD

**Issue by Graded5** [space-wizards/space-station-14#28878]
## Description
We recently tried to make emagged borgs harder to discover, but in the process made it 10 times easier. When a borg gets emagged, it's lock breaks, meaning if you want to catch a borg all you have to do is alt click it and see if you, for reason, have access.

**Reproduction**
- Emag a borg, making sure to unlock the panel first
- Close the panel and lock it
- Take off your ID
- Observe how you can still lock and unlock the borg without the proper accesses

**Screenshots**

**Additional context**
As much as I hate this, I do think that science needs *some* way of figuring out if a borg is emagged or not. Its not fun to kill borgs because they *might* be malfunctioning and its not fun to be the borg that isnt emagged but was still killed because a scientist had an emag.

**Comment by EmoGarbage404** [comment]
its just a boolean on the lockcomponent so you can just set it to false

**Comment by Plykiya** [comment]
issue solved, can close it now

**Comment by PJB3005** [comment]
The PR didn't work, this is still an issue.

**Comment by SurrealShibe** [comment]
this is still a problem. it is very easy to catch emagged borgs this way and offers no way to rp out of it as a borg

**Comment by SurrealShibe** [comment]
if anyone has ideas for a solution, please implement or post them here

**Comment by OrbitSystem07** [comment]
> this is still a problem. it is very easy to catch emagged borgs this way and offers no way to rp out of it as a borg

I do feel the need to point out this only happens if the borg is opened with an access breaker, which... makes sense, given what an access breaker does

**Comment by SurrealShibe** [comment]
yeah, i suppose, it just is really unfun when someone walks by, does a single alt-click, and then has a reason to gib you. it also doesn't help that you can be access broken but not emagged, like if you ran away during the process, in which case science is down like 50 steel. 
the reason it's so hard to come up with a solution is because the current behavior is consistent with how one expects the access breaker to work

**Comment by SurrealShibe** [comment]
maybe if a small do-after was added to lock/unlock attempts on borgs?

**Comment by slarticodefast** [comment]
Emag and access breaker are two separate devices now, so this is no longer an issue.
